extension-manifest-v3/scripts/build.js

```diff
@@ -183,6 +183,20 @@ if (manifest.browser_action?.default_popup) {
   source.push(manifest.browser_action.default_popup);
 }
 
+// offscreen documents
+if (
+  manifest.permissions.includes('offsceen') ||
+  manifest.optional_permissions.includes('offsceen')
+) {
+  readdirSync(join(options.srcDir, 'pages', 'offscreen'), {
+    withFileTypes: true,
+  })
+    .filter((dirent) => dirent.isDirectory() && !dirent.name.startsWith('.'))
```

Comment:
What should be filtered by starsWith('.')? Is it only to exclude `.` and `..`?

(I'm not sure if it specified if you get directories like this `path/...` or that `./path/...` without normalizing. That might depend on factors like the OS and the current working directory.)

Reply:
it's meant to ignore "hidden" folders. `..` is not returned by `readdirSync`